### DISCUSSION TITLE
Download temperature data in bulk from website

### DISCUSSION THREAD

**Discussion by christophe-rd** [Homebrew/discussions#3495] [Getting started]
Hi,

I've never used Homebrew before. Therefore,  I have no idea how it works.
I need it to export data. 
Here are the steps to get the data from home brew:

https://drive.google.com/drive/folders/1avYVLKfYvJTP_-uWxpNpbimg8dBlZhcM

My specific code, that I adapted from the example is the following:
```
for annee in `seq 2004 2022`;do for mois in `seq 1 12`;do wget --content-disposition
"https://climat.meteo.gc.ca/climate_data/bulk_data_f.html?format=csv&stationID= 7086720&Year=${anne
e}&Month=${mois}&Day=14&timeframe=2&submit=++T%C3%A9l%C3%A9charger+%0D%0Ades+donn%
C3%A9es" ;done;done
```
I know I'm not of huge help in the indications, but I'm quite lost. 
If someone could help me with that, it would be fantastic !

**Comment by carlocab** [comment]
Have you installed Homebrew? If you have not, install Homebrew using the instructions as https://brew.sh. Then, do
```
brew install wget
```
Finally, you can run the code snippet you show above (assuming it is well-formed code; I haven't checked).

You may wish to write to ec.services.climatiques-climate.services.ec@canada.ca to tell them that the README that they provided you does not provide enough details to allow you to do what you need.

**Comment by mtthwborg** [comment]
Removing spaces that shouldn't be there and may have arisen from copying and pasting worked for me. I.e. try:

for annee in `seq 2004 2022`;do for mois in `seq 1 12`;do wget --content-disposition "https://climat.meteo.gc.ca/climate_data/bulk_data_f.html?format=csv&stationID= 7086720&Year=${annee}&Month=${mois}&Day=14&timeframe=2&submit=++T%C3%A9l%C3%A9charger+%0D%0Ades+donn%C3%A9es" ;done;done
